### ISSUE TITLE
Improving reproducibility of notebook-generated webpages (i.e., more similar to workflowr)

### ISSUE THREAD

**Issue by pcarbo** [stephenslab/ipynb-website#7]
We can borrow some ideas from [Reproducible](https://github.com/sgravel/reproducible):

> Reproducible effectively creates a one-to-one mapping between commit hashes and experimental output data.

@gaow Maybe we can return to this at some point.

Thanks to [Simon Gravel](https://github.com/sgravel) for sharing.

**Comment by gaow** [comment]
Good idea and simple concept, and thanks to Simon's code we only have interface to worry about ... which is non-trivial :) we can return to the interface design later.
